### DISCUSSION TITLE
Maybe migrate from unpkg to jsdelivr

### DISCUSSION THREAD

**Discussion by dluccheseFmoq** [buefy/buefy#4033] [Ideas]
unpkg seem to be reliable no more. We got an outage today because of it. We previously - many months ago - had an outage with a buggy release on unpkg so we downloaded the minified version of Buefy on our website.
Because of the Buefy documentation sample however, we still fetched vue.js from unpkg. This morning unpkg was down for a few hours and people could not access our websites. 

We should at least use the same link as in vue 2 documentation to fetch vue.js in the documentation:
https://v2.vuejs.org/v2/guide/installation#CDN

Ideally migrate or add Buefy to jsdelivr or another more reliable CDN.

Doc on the unpkg issue:
https://github.com/mjackson/unpkg/issues/383

**Comment by kikuomax** [comment]
@dluccheseFmoq Thanks for sharing the issue of unpkg. jsDelivr says they can deliver any package on npm. Actually, the following URL worked without any configuartion: <https://cdn.jsdelivr.net/npm/buefy@0.9.29/dist/buefy.min.js>
